src/scss/components/_login-options.scss

```diff
@@ -16,3 +16,8 @@
     @include margin(auto);
   }
 }
+
+.#{prefix(login-options__cosign)} {
```

Comment:
this belongs in the code block above, instead of making these BEM classnames manually, you should use `@include bem.element('cosign')` instead and define the style rules in there.